modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/workflow/mgt/WorkflowManagementTestCase.java

```diff
@@ -1334,6 +1345,177 @@ public void testRemoveBPSProfile() {
         }
     }
 
+    @BeforeGroups("ListWorkflowsTests")
+    public void createWorkflowsForListing() throws RemoteException, WorkflowAdminServiceWorkflowException {
+
+        if (workflowId != null && isWorkflowPresent(workflowId)) {
```

Comment:
Better to use `StringUtils.isNotEmpty` instead of null check. Check other places also